table/server/underdb/glue/src/main/java/alluxio/table/under/glue/GlueDatabase.java

```diff
@@ -21,7 +21,7 @@
 import alluxio.table.common.UdbPartition;
 import alluxio.table.common.layout.HiveLayout;
 import alluxio.table.common.udb.PathTranslator;
-import alluxio.table.common.udb.UdbBypassSpec;
+import alluxio.table.common.udb.UdbFilterSpec;
```

Comment:
keep each import section alphabetically

Reply:
Optimized and sorted imports in d528def944255c58bb67779fb96c016afb41c5e7